kind-istio/workload/deploy-nginx.yaml

```diff
@@ -0,0 +1,44 @@
+##################################################################################################
+# nginx dummy deployment
```

Comment:
Quick question: why do we need this?

Reply:
The wasm module is deployed on the proxy of the nginx workload and the e2e tests will try to connect to this nginx. The alternative could be deploy the waf just in front of the ingress and then making a request that would go nowhere maybe? I think is okay to have a service that we are trying to reach/protect